pyspecdata/DCCT_function.py

```diff
@@ -199,6 +202,15 @@ def DCCT(
             ax_list.append(
                 plt.axes([LHS_labels + LHS_pad, b, width, axes_height])
             )  # lbwh
+    dx = LHS_labels + LHS_pad  # Fig coord
+    axis_to_figure = ax_list[0].transAxes + fig.transFigure.inverted()
+    _, dy = axis_to_figure.transform(r_[0, 0])
+    total_scale_transform = IdentityTransform() + ScaledTranslation(
+        dx, dy, fig.transFigure
+    )
+    total_trans = blended_transform_factory(
+        total_scale_transform, fig.transFigure
+    )
```

Comment:
So line 208 to hear is what I am describing in my long comment. These are the only two coordinate systems that you should need.